requirements.txt

```diff
@@ -1,6 +1,17 @@
+#   NOTE: this is a mess, because some things are required on provider, some
+#         things are required on requestor and things "really required on provider"
+#         are different from those required for echo-server-based tests.
+#         I don't fix it now because this will change either way when we have a final interface.
```

Comment:
We need at least 4 separate requirement-concepts:
* requirements that should be installed in provider image for the echo_server to work (e.g. testing)
* way of declaring provider requirements for any other app than echo_server
* requirements for requestor-provider communication
* requirements for requestor-running catchall server (this could be together with the previous one)

I don't think this is a good moment to prepare this because of expected http-requestor-proxy evolution.